cli/internal/transformerpipeline/pipeline.go

```
@@ -82,6 +87,13 @@ func (lp *TransformerPipeline) OnOutput(fn func([]byte) error) error {
 }
 
 func (lp *TransformerPipeline) Close() error {
+	// Closing the pipeline happens on both source as well as destination close.
+	// Not handling this will result in a close of closed channel panic.
+	if lp.clientWrappers[0].closed {
+		return nil
+	}
+	lp.clientWrappers[0].closed = true
```

I'd `ClosedSend` before setting this to `true`, and ignore the error.